lighthouse-plugin-ad-speed-insights/audits/static-ad-tags.js

```diff
@@ -99,8 +119,19 @@ class StaticAdTags extends Audit {
     const dynamicReq = tagReqs.find((r) => !isStaticRequest(r));
     const table = dynamicReq ? getDetailsTable(dynamicReq) : [];
 
+    const failed = numStatic < numTags;
+    let displayValue;
```

Comment:
Does this work fine as undefined rather than an empty string?

Reply:
Seems like it, but changed it to the empty string to be safe